lib/prosperworks/company.rb

```diff
@@ -1,5 +1,6 @@
 module ProsperWorks
   class Company < BaseEntity
+    extend ApiOperations::Search
```

Comment:
You should add this line to `BaseEntity` instead so we don't have to add it to each individual file